ext/sqlite3/tests/bug66550.phpt

```diff
@@ -15,7 +15,11 @@ $stmt = $db->prepare('SELECT bar FROM foo WHERE id=:id');
 // Close the database connection and free the internal sqlite3_stmt object
 $db->close();
 // Access the sqlite3_stmt object via the php_sqlite3_stmt container
-$stmt->reset();
+try {
+    $stmt->reset();
+} catch (\Error $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
 ?>
---EXPECTF--
-Warning: SQLite3Stmt::reset(): The SQLite3 object has not been correctly initialised in %s
+--EXPECT--
+The SQLite3 object has not been correctly initialised
```

Comment:
This message doesn't match the test well. I would at least add `or already closed` to it.